src/include/platform/CHIPDeviceConfig.h

```diff
@@ -91,6 +91,52 @@
 #define CHIP_DEVICE_CONFIG_MAX_EVENT_QUEUE_SIZE 100
 #endif
 
+/**
+ * CHIP_DEVICE_CONFIG_ENABLE_BG_EVENT_PROCESSING
+ *
+ * Enable support for background event processing.
+ */
+#ifndef CHIP_DEVICE_CONFIG_ENABLE_BG_EVENT_PROCESSING
+#define CHIP_DEVICE_CONFIG_ENABLE_BG_EVENT_PROCESSING 1
```

Comment:
Please disable by default on most platforms, otherwise there is a significant memory penalty.

Reply:
Sure, it was only on because I was testing before uploading the commit.